draft-ietf-webtrans-http3.md

```diff
@@ -366,11 +366,50 @@ an implementation to choose what stream or datagram to discard.
 
 # Session Termination
 
-An WebTransport session over HTTP/3 is terminated when either endpoint closes
-the stream associated with the CONNECT request that initiated the session.
+A WebTransport session over HTTP/3 is considered terminated when either of the
+following conditions is met:
+
+* the CONNECT stream is closed, either cleanly or abruptly, on either side; or
+* a CLOSE_WEBTRANSPORT_SESSION capsule is either sent or received.
+
 Upon learning about the session being terminated, the endpoint MUST stop
 sending new datagrams and reset all of the streams associated with the session.
 
+To terminate a session with a detailed error message, an application MAY send
+an HTTP capsule {{HTTP-DATAGRAM}} of type CLOSE_WEBTRANSPORT_SESSION (0x2843).
+The format of the capsule SHALL be as follows:
+
+~~~
+CLOSE_WEBTRANSPORT_SESSION Capsule {
+  Application Error Code (32),
+  Application Error Message (..8192),
+}
```

Comment:
This would match HTTP-DATAGRAM better:
```suggestion
CLOSE_WEBTRANSPORT_SESSION Capsule {
  Type (i) = CLOSE_WEBTRANSPORT_SESSION,
  Length (i),
  Application Error Code (32),
  Application Error Message (..8192),
}
```